cmd/prometheus-config-reloader/main.go

```diff
@@ -106,6 +112,12 @@ func main() {
 		stdlog.Fatal(err)
 	}
 
+	err = web.Validate(*webConfig)
+	if err != nil {
+		level.Error(logger).Log("msg", "Unable to validate web configuration file. err: "+err.Error())
```

Comment:
(nit) better to have a separate key for the error
```suggestion
		level.Error(logger).Log("msg", "Unable to validate web configuration file", "err", err)
```